setup.cfg

```diff
@@ -6,3 +6,14 @@ ignore = E203, E266, E501, W503
 max-line-length = 110
 max-complexity = 18
 select = B,C,E,F,W,T4,B9
+
+[mypy]
+ignore_missing_imports = true
+warn_unused_configs = True
+no_implicit_optional = True
+warn_redundant_casts = True
+warn_unused_ignores = True
+show_error_context = True
+show_column_numbers = True
+show_error_codes = True
+pretty = True
```

Comment:
I'd like to keep this harmonized with marshmallow/webargs/... as much as possible.

https://github.com/marshmallow-code/marshmallow/blob/dev/setup.cfg

```
[mypy]
ignore_missing_imports = true
warn_unreachable = true
warn_unused_ignores = true
warn_redundant_casts = true
no_implicit_optional = true
```